http.md

```diff
@@ -85,6 +81,19 @@ app.use((err, req, res, next) => {
 });
 ```
 
+## Setting Timeouts
+
+By default, all HTTP requests will timeout at 29 seconds. This can be raised or lowered using the `http.setTimeout` method. For example, to set the timeout to 10 seconds:
+
+```javascript
+http.setTimeout(10000); // 10 seconds
+http.node.use(expressApp);
+```
+
+!!! note
+You must assure Response Streaming is enabled in your app's settings to raise the timeout above 29 seconds. This is enabled by default. You can check in your environment's Settings tab.
```

Comment:
"You must have Response Streaming enabled..."

"assure" is the wrong usage here.

Reply:
Yep, meant "ensure" lol. Still on first coffee...